### PR TITLE
[TIMOB-23627] iOS10: Throw error when not having the required iOS 10 plist-keys

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-23627

### CODE REVIEW

File: apidoc/Titanium/Media/Media.yml [hunk 1]
```diff
@@ -298,13 +303,41 @@ methods:
...
+        grants or denies permission, the system remembers the choice for future use in the same app. 
+        If permission is not granted, or if the user has not yet responded to the permission prompt, 
+        any audio recording sessions record only silence.
+
+        In addition, Apple introduced the Info.plist key `NSMicrophoneUsageDescription` that is used to display an
+        own description while authorizing microphone permissions. In iOS 10, this key is mandatory and the application
+        will crash if your app does not include the key. Check the [Apple docs](https://developer.apple.com/library/prerelease/content/documentation/General/Reference/InfoPlistKeyReference/Articles/CocoaKeys.html) 
+        for more information.
+    platforms: [iphone, ipad]
+    since: 5.1.0
```
Comment (AngelkPetkov) [inline apidoc/Titanium/Media/Media.yml]: Change to 6.0.0

Comment (hansemannn) [inline apidoc/Titanium/Media/Media.yml]: No, the API was already there but not documented.

Comment (AngelkPetkov) [inline apidoc/Titanium/Media/Media.yml]: Ahh okay, 👍

**Comment by AngelkPetkov** [comment]
FT and CR approved!

**Pull request merged**